addons/website/views/snippets/s_tabs.xml

```diff
@@ -2,67 +2,62 @@
-                                <li class="nav-item">
-                                    <a class="nav-link"
-                                        id="nav_tabs_link_2"
-                                        data-toggle="tab"
-                                        href="#nav_tabs_content_2"
-                                        role="tab"
-                                        aria-controls="nav_tabs_content_2"
-                                        aria-selected="false">Profile</a>
-                                </li>
-                                <li class="nav-item">
-                                    <a class="nav-link"
-                                        id="nav_tabs_link_3"
-                                        data-toggle="tab"
-                                        href="#nav_tabs_content_3"
-                                        role="tab"
-                                        aria-controls="nav_tabs_content_3"
-                                        aria-selected="false">Contact</a>
-                                </li>
-                            </ul>
+            <div class="s_tabs_wrap">
+                <div class="s_tabs_main">
+                    <div class="s_tabs_nav mb-3">
+                        <ul class="nav nav-pills justify-content-center" role="tablist">
+                            <li class="nav-item">
+                                <a class="nav-link active"
+                                    id="nav_tabs_link_1"
+                                    data-toggle="tab"
+                                    href="#nav_tabs_content_1"
+                                    role="tab"
+                                    aria-controls="nav_tabs_content_1"
+                                    aria-selected="true">Home</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link"
+                                    id="nav_tabs_link_2"
+                                    data-toggle="tab"
+                                    href="#nav_tabs_content_2"
+                                    role="tab"
+                                    aria-controls="nav_tabs_content_2"
+                                    aria-selected="false">Profile</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link"
+                                    id="nav_tabs_link_3"
+                                    data-toggle="tab"
+                                    href="#nav_tabs_content_3"
+                                    role="tab"
+                                    aria-controls="nav_tabs_content_3"
+                                    aria-selected="false">Contact</a>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="s_tabs_content tab-content">
```

Comment:
Actually there is a lot more to think about here. Is this new class needed? If yes, the option won't work with old snippets so this snippet have to marked as incompatible somehow, vcss? vxml?